runtime/runtime_options_manager.go

```diff
@@ -74,3 +76,39 @@ func (w *optionsManager) RegisterListener(
 func (w *optionsManager) Close() {
 	w.watchable.Close()
 }
+
+// NewNoOpOptionsManager returns a no-op options manager that cannot
+// be updated and does not spawn backround goroutines (useful for globals
+// in test files).
+func NewNoOpOptionsManager(opts Options) OptionsManager {
```

Comment:
lol this came from annoyance eh?

Reply:
Yeah `leaktest.(...)` started going off 100% each test run due to a lingering goroutine from `func init` in the `storage` package that registered a listener that never closes.  To avoid that weirdness put this together.

Reply:
much appreciated. i've always chased the different leaks, this is cleaner